extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.tsx

```diff
@@ -93,8 +89,17 @@ function TrackedCornerstoneViewport(props) {
     };
   }, [isTracked]);
 
-  if (trackedSeries.includes(SeriesInstanceUID) !== isTracked) {
-    setIsTracked(!isTracked);
+  // A current image id will only exist for viewports that can have measurements tracked.
+  // Typically these are stack viewports and those volume viewports for the series of acquisition.
+  const currentImageId = cornerstoneViewportService
+    .getCornerstoneViewport(viewportId)
+    ?.getCurrentImageId();
+  if (currentImageId) {
```

Comment:
how about 

```js
if (currentImageId && trackedSeries.includes(SeriesInstanceUID) !== isTracked) {
  setIsTracked(!isTracked);
}
```

Reply:
No because if we have a `currentImageId` and tracking is on and has NOT changed then we do NOT want to simply to turn tracking off.